source/user-manual/reference/internal-options.rst

```diff
@@ -1007,6 +1007,18 @@ Vulnerability Detection
 +                                                    +----------------+----------------------------------------------------------------------------------+
 |                                                    | Allowed values | Any integer between 1 and 100000                                                 |
 +----------------------------------------------------+----------------+----------------------------------------------------------------------------------+
+|  **vulnerability-detection.remediation_lru_size**  | description    | lru cache size assigned for remediations' os data (in number of elements).       |
++                                                    +----------------+----------------------------------------------------------------------------------+
+|                                                    | default value  | 2048                                                                             |
++                                                    +----------------+----------------------------------------------------------------------------------+
+|                                                    | allowed values | any integer between 1 and 100000                                                 |
++----------------------------------------------------+----------------+----------------------------------------------------------------------------------+
+|  **vulnerability-detection.disable_manager_scan**  | description    | Disable or enable the vulnerability-detection in the manager.                    |
++                                                    +----------------+----------------------------------------------------------------------------------+
+|                                                    | default value  | 0 (Enabled)                                                                      |
++                                                    +----------------+----------------------------------------------------------------------------------+
+|                                                    | allowed values | 1 to Disable and 0 to Enable.                                                    |
++----------------------------------------------------+----------------+----------------------------------------------------------------------------------+
```

Comment:
We are not going to document this (I've closed the associated issue)
```suggestion
```

Reply:
On it!